sqljocky/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2017, teja.
```

Comment:
Use the generic Jaguar license: https://github.com/Jaguar-dart/jaguar/blob/master/jaguar/LICENSE